module/Organizations/src/Organizations/Repository/Filter/PaginationQuery.php

```diff
@@ -55,28 +67,64 @@ public function __construct($auth)
      */
     public function createQuery($params, $queryBuilder)
     {
-        $userID = $this->auth->getUser()->getId();
         if ($params instanceof Parameters) {
             $value = $params->toArray();
         } else {
             $value = $params;
         }
 
-        if (isset($params['q']) && $params['q']) {
+        if (isset($params['q']) && $params['q'] && $params['q'] != 'en/organizations/profile' ) {
             $queryBuilder->text($params['q'])->language('none');
         }
-        
-        /*
-         * We only show organizations to which the user has view permissions.
-         */
-        $queryBuilder->field('permissions.view')->equals($userID);
```

Comment:
If this filter is used for the organizations list of a recruiter, we want of course only list organizations, the recruiter has view permissions for.